app/src/components/encryptionService.js

```diff
@@ -0,0 +1,166 @@
+const config = require('config');
+const crypto = require('crypto');
+
+const SERVICE = 'EncryptionService';
+
```

Comment:
Some of the code in this file appears to be taken from a [gist](https://gist.github.com/AndiDittrich/4629e7db04819244e843), and one concern is that if a vulnerability in the gist code is found, then we would be unaware of problem unless we periodically check the gist for updates. Plus how do we know that the code is correct? I couldn't find a "standard" package offered by some big reliable company that we could use, but what about [cryptr](https://www.npmjs.com/package/cryptr)? It seems to be maintained and widely used (for better or worse!).

Reply:
the `crypto` lib itself is standard and that's how you use it. if you look at `cryptr` it's basically the same code. So up to you if you want the lib included (it won't introduce a bunch of other dependencies so that's good).